packages/@okta/vuepress-site/guides/sign-into-mobile-app/before-you-begin/index.md

```diff
@@ -1,17 +1,16 @@
 ---
-title: Sign Users in to Your Mobile App
+title: Before You Begin
 ---
-This guide shows you how to use Okta as the user store for your mobile application and sign users in to your app.
+This guide shows you how to use Okta as the user store for your mobile application. When you've finished following the steps, your app will open a browser to sign users in.
```

Comment:
Removed future tense:
This guide shows you how to use Okta as the user store for your mobile application. When you've finished following the steps, your app opens a browser to sign users in.